x-pack/plugins/uptime/public/components/monitor_management/monitor_list/monitor_list.tsx

```diff
@@ -66,14 +66,19 @@ export const MonitorManagementList = ({
         defaultMessage: 'Monitor name',
       }),
       render: ({
-        attributes: { name },
+        attributes: { name, type },
         id,
       }: {
         attributes: Partial<MonitorFields>;
         id: string;
       }) => (
         <EuiLink
-          href={`${basePath}/app/uptime/monitor/${Buffer.from(id, 'utf8').toString('base64')}`}
+          href={`${basePath}/app/uptime/monitor/${Buffer.from(
+            /* Monitor Management currently only supports inline browser monitors.
+             * Inline browser monitors append `inline` to the monitor id */
+            `${id}${type === DataStream.BROWSER ? `-inline` : ''}`,
```

Comment:
i thought we are going to handle this in heartbeat https://github.com/elastic/beats/issues/29505
But if you doing this as a makeshift solution. I am fine by it.

Reply:
@shahzad31 That only accounts for removing the appended ` - Inline` in the journey title, not the id, or at least it should. We already opened pandora's box with this id scheme, we can't change it now or we'll lose the history for these monitors.